src/syntax/syntax.rkt

```diff
@@ -126,55 +163,64 @@
   [libm acos acosf] [bf bfacos] [cost 90]
   [->c/double (curry format "acos(~a)")]
   [->c/mpfr (curry format "mpfr_acos(~a, ~a, MPFR_RNDN)")]
-  [->tex (curry format "\\cos^{-1} ~a")])
+  [->tex (curry format "\\cos^{-1} ~a")]
+  [nonffi (match-lambda [(list x) (flacos x)])])
```

Comment:
Hmm. `flacos` is the version that takes only floating-point numbers; `acos` takes exact numbers as well and does the exactness tracking. I think we want that second version.

Reply:
Using the real versions of functions comes with a massive performance hit. When doing a test using `sin` instead of `flsin` on `(FPCore (a) (sin a))`, my computer runs out of memory (8 GB) before it outputs and answer. Is there a better way to do this than just a drop-in replacement?